app/assets/javascripts/discourse/tests/acceptance/do-not-disturb-test.js

```diff
@@ -48,6 +48,33 @@ acceptance("Do not disturb", function (needs) {
     );
   });
 
+  test("Can be invoked via keyboard", async function (assert) {
+    updateCurrentUser({ do_not_disturb_until: null });
+
+    await visit("/");
+    await click(".header-dropdown-toggle.current-user");
+    await click(".menu-links-row .user-preferences-link");
+
+    await click(".do-not-disturb");
+
+    assert.ok(exists(".do-not-disturb-modal"), "modal to choose time appears");
+
+    let tiles = queryAll(".do-not-disturb-tile");
+    assert.ok(tiles.length === 4, "There are 4 duration choices");
```

Comment:
```suggestion
    assert.strictEqual(tiles.length, 4, "There are 4 duration choices");
```

The diff will be more useful when the assertion fails.